### ISSUE TITLE
Bug: Buttons do not have an accessible name

### ISSUE THREAD

**Issue by priyankarpal** [pbclife/gitopener.vercel.app#151]
### Is there any existing issue for this?

- [X] I have searched the existing issues

### Description

When a button doesn't have an accessible name, screen readers announce it as a "button", making it unusable for users who rely on screen readers.

### Screenshots

![image](https://user-images.githubusercontent.com/88102392/232234646-cd3fd825-11e2-41b0-b399-71f1fb6b462a.png)

**Comment by MutiatBash** [comment]
@priyankarpal Hello…..is this issue still available to work on….I’d like to work on it

**Comment by priyankarpal** [comment]
> @priyankarpal Hello…..is this issue still available to work on….I’d like to work on it

yeah assigned to you

**Comment by MutiatBash** [comment]
@priyankarpal ……I’m having difficulties setting up the project, I’m getting errors when I run yarn install….regarding sharp installation

**Comment by priyankarpal** [comment]
> @priyankarpal ……I’m having difficulties setting up the project, I’m getting errors when I run yarn install….regarding sharp installation

@sboy99

**Comment by MutiatBash** [comment]
![105E8887-BABC-4520-AB31-0E0BE0FEC4A9](https://github.com/pbclife/gitopener.vercel.app/assets/108807732/12503798-531d-47f9-a734-dccd79d6b536)

this is the error message I get and i tried installing sharp separately as well

**Comment by priyankarpal** [comment]
> ![105E8887-BABC-4520-AB31-0E0BE0FEC4A9](https://user-images.githubusercontent.com/108807732/248309828-12503798-531d-47f9-a734-dccd79d6b536.jpeg)
> 
> this is the error message I get and i tried installing sharp separately as well

but I'm not getting any error
